utils/syncthing/Makefile

```diff
@@ -0,0 +1,66 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=syncthing
+PKG_VERSION:=1.2.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=syncthing-source-v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/syncthing/syncthing/releases/download/v$(PKG_VERSION)
+PKG_HASH:=0339877effdcf3bf8aa7d4d1e50b878992792e4752ff778f27788bf71eccecd0
+
+PKG_MAINTAINER:=Paul Spooren <mail@aparcar.org>
+PKG_LICENSE:=MPL-2.0
+PKG_LICENSE_FILES:=LICENSE
+PKG_CPE_ID:=cpe:/a:syncthing:syncthing
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+PKG_UNPACK:=$(TAR) -C $(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
```

Comment:
Not sure, if I like this. Shouldn't be PKG_BUILD_DIR working if set correctly?
@jefferyto thoughts?

Reply:
The source tarball (`syncthing-source-v$(PKG_VERSION).tar.gz`) extracts to `syncthing` (no version number), so I think this `PKG_UNPACK` is ok.

Reply:
It's a matter of style honestly. Can also be done with

```
PKG_BUILD_DIR=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)/$(PKG_NAME)
```

Reply:
Oh this looks better... but more typing when I'm crawling inside build_dir to do debugging :laughing:

Reply:
Thanks, I'm always happy to learn new tricks...